### PR TITLE
111 blog post teal release in cran

### LINKED ISSUE #111: Blog Post:  teal release in CRAN
### Blog Post

I'd like to write a post about teal availability in CRAN and what user needs to do to if they're interested to upgrade their teal apps.

### PR DESCRIPTION
**Pull request by donyunardi**
Close #111 

In addition to blog post, I added the new `teal` logo to the media folder.

### CODE REVIEW

File: posts/2024-02-13_teal_on_cran/teal_on_cran.qmd [hunk 1]
```diff
@@ -25,28 +25,28 @@ We're thrilled to announce that `teal` v0.15.0 has been released on CRAN!
 
 This marks a significant milestone in our journey, and we're incredibly excited about the possibilities teal brings to the R community, particularly within clinical trial settings.
```
Comment (pawelru) [inline posts/2024-02-13_teal_on_cran/teal_on_cran.qmd:26]:
```suggestion
This marks a significant milestone in our journey, and we're incredibly excited about the possibilities `teal` brings to the R community, particularly within clinical trial settings.
```

File: posts/2024-02-13_teal_on_cran/teal_on_cran.qmd [hunk 2]
```diff
@@ -25,28 +25,28 @@ We're thrilled to announce that `teal` v0.15.0 has been released on CRAN!
...
 
-One of the most notable changes in this release is the introduction of `teal_data`. This addition enhances how data is handled within the `teal` framework, paving the way for custom data modules tailored to the needs of our R programmers. With `teal_data`, users can expect improved efficiency and flexibility in managing their data, opening doors to innovative approaches in data analysis and visualization.
+One of the most notable changes in this release is the introduction of `teal_data`. This addition enhances how data is handled within the `teal` framework, paving the way for custom data modules tailored to the needs of our R users, both inside and outside the clinical trial space. With `teal_data`, users can expect improved efficiency and flexibility in managing their data, opening doors to innovative approaches in data analysis and visualization.
```
Comment (pawelru) [inline posts/2024-02-13_teal_on_cran/teal_on_cran.qmd:28]:
```suggestion
One of the most notable changes in this release is the introduction of `teal_data` class. This addition enhances how data is handled within the `teal` framework, paving the way for custom data modules tailored to the needs of our R users, both inside and outside the clinical trial space. With `teal_data`, users can expect improved efficiency and flexibility in managing their data, opening doors to innovative approaches in data analysis and visualization.
```

**Comment by pawelru** [comment]
small corrections but all good! I'll approve it for you to merge it promptly

**Review by pawelru — APPROVED** [review]

**Comment by bms63** [comment]
@donyunardi Could we shrink the teal hex down slightly?

image

**Comment by bms63** [comment]
@donyunardi could we also get a screenshot of one the apps in action?  They are very beautiful and streamlined - nice for folks to see it.

Also, I don't the see the teal pkgdown site is linked anywhere - could that be included as well?

**Comment by donyunardi** [comment]
@bms63 Thank you for the feedback.

Image updated
Screenshot 2024-02-14 at 9 30 43 AM

and I added link to teal's pkgdown and screenshot as suggested.

**Comment by bms63** [comment]
LGTM! I'll make a quick announcement tmr morning!!

**Pull request merged**